docs/rules/func-style.md

```diff
@@ -46,6 +46,26 @@ Due to these different behaviors, it is common to have guidelines as to which st
 
 This rule is aimed at enforcing a particular type of function style throughout a JavaScript file, either declarations or expressions. You can specify which you prefer in the configuration.
 
+### Options
+
+```json
+"func-style": [2, "expression"]
+```
+
+This is the default configuration.  It reports an error when function declarations are used instead of function expressions.
```

Comment:
I'm not sure the docs are consistent enough to say "every where else", but it is more common and does make more sense. :wink: